Formula/q/qt.rb

```diff
@@ -217,6 +218,8 @@ def install
       # `___builtin_available` to ensure compatibility.
       config_args << "-skip" << "qtwebengine" if DevelopmentTools.clang_build_version <= 1200
     else
+      cmake_args << "-DQT_FEATURE_xcb=ON"
```

Comment:
Just noting that this probably won't do anything. I noticed that the `./configure` script is "smart" where it will automatically override variable if it doesn't autodetect it. This is the case for both `-D` cmake args and `-feature...` configure args.

We would have to switch to directly running `cmake` instead of `./configure` to bypass this behavior.